server/dist/ssr-template.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style type="text/css">
```

Comment:
Would styles normally be in a separate file?

Reply:
yes, good call.  I will refactor the styles out to a separate file.